src/web/mjs/connectors/ToonilyMe.mjs

```diff
@@ -0,0 +1,31 @@
+import MadTheme from './templates/MadTheme.mjs';
+
+export default class ToonilyMe extends MadTheme {
+    constructor() {
+        super();
+        super.id = 'toonilyme';
+        super.label = 'ToonilyMe Manga';
```

Comment:
i believe i have removed the "Manga" myself because i didnt want to bother you with that.
Please accept reviewer modifications :D